superset/assets/src/visualizations/deckgl/utils.js

```diff
@@ -6,17 +6,14 @@ import { hexToRGB } from '../../modules/colors';
 export function getBreakPoints({
     break_points: formDataBreakPoints,
     num_buckets: formDataNumBuckets,
-    metric,
-  }, features) {
+  }, features, accessor) {
   if (!features) {
     return [];
   }
   if (formDataBreakPoints === undefined || formDataBreakPoints.length === 0) {
     // compute evenly distributed break points based on number of buckets
-    const numBuckets = formDataNumBuckets
-      ? parseInt(formDataNumBuckets, 10)
-      : 10;
-    const [minValue, maxValue] = extent(features, d => d[metric]);
+    const numBuckets = formDataNumBuckets ? parseInt(formDataNumBuckets, 10) : 10;
```

Comment:
nit: Better to leave a `DEFAULT_NUM_BUCKETS = 10` at the top of the file. Can we replace all the inline conditions in this file for better readability/consistency